entry_types/scrolled/package/src/frontend/index.js

```diff
@@ -21,6 +21,8 @@ export * from './InlineCaption';
 export * from './Text';
 export * from './Video';
 
+export * from './MediaPlayer';
```

Comment:
Now that `MediaPlayer` only takes sources, I think it would make sense to introduce a `VideoPlayer` component (or rewrite the existing `Video` component) so that we still have a component that takes a video file as returned by `useFile`, comes up with the correct sources and renders a `MediaPlayer`.

Reply:
it can be handled by passing type as a prop

Reply:
As [mentioned earlier](https://github.com/codevise/pageflow/pull/1398#discussion_r413558763) there will be more logic involved for assembling the list of sources. This logic needs to live somewhere and it should not be in the individual clients using the `MediaPlayer`. I therefore still think a video specific component would make sense to already add now. It will also make adding stories easier, since there we also have to deal with `VideoFile` objects from seed date instead of just source arrays.

Reply:
added VideoPlayer and AudioPlayer in pageflow-scrolled/frontend